src/s1reader/s1_orbit.py

```diff
@@ -170,10 +241,8 @@ def get_orbit_dict(mission_id, start_time, end_time, orbit_type):
         err_msg = f'{orbit_type} not a valid orbit type'
         raise ValueError(err_msg)
 
```

Comment:
Can't comment on the docstring but maybe we should change it now from 

```python
    start_time: datetime object
        Sentinel start acquisition time
    end_time: datetime object
        Sentinel end acquisition time
```

to

```python
    start_time: datetime object
        Starting time for search query
    end_time: datetime object
        Ending time  for search query
```

since this function really doesn't care about sensing times; it just makes the query for whatever you give (which in our case is not the start/stop senting time

Reply:
Good catch. The docstring needs to be revised.